<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersDialog</class>
 <widget class="QDialog" name="ParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButton_global">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Global Parameter</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Partionned Parameter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_validate">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Validate</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>41</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>42</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Partition #  </string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>8</number>
   </property>
   <property name="minimum">
    <double>-1000000000000000009248546019891598444566210341657546615907521388633406505708118389308454908642502206536081877044340989143693798086218131232373875663313958712699944969706504756133888.000000000000000</double>
   </property>
   <property name="maximum">
    <double>1000000000000000048346692115553659057528394845890514255872.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Value :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>released()</signal>
   <receiver>ParametersDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>released()</signal>
   <receiver>ParametersDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
